utils.go

```diff
@@ -131,3 +135,52 @@ func IsGenerated(path string, content []byte) bool {
 
 	return false
 }
+
+func init() {
```

Comment:
It would be very convenient for the future maintainers if the essense of the optimization applied here (logic behind rewriting a regexp) is documented here, on the high level. 🙏